articles/azure-arc/servers/quickstart-onboard-powershell.md

```diff
@@ -49,6 +49,13 @@ Id                    : 5be92c87-01c4-42f5-bade-c1c10af87758
 Type                  :
 ```
 
+>>Note: 
+It may take a while to get your SPN permissions properly populated. Running the following role assignment to set the permissions much faster.
```

Comment:
I suggest removing the note and just telling them to always run this, it's simpler